cellpack/autopack/Compartment.py

```diff
@@ -3120,6 +3119,76 @@ def getSurfaceInnerPoints(self, boundingBox, spacing, display=True, useFix=False
         self.grid_distances = distances
         return insidePoints, surfacePoints
 
+    def create_voxelized_mask(
+        self,
+        x_width,
+        y_width,
+        z_width,
+        center,
+        voxel_size,
+        mesh_store,
+        hollow=False,
+    ):
+        """
+        Creates a mask of the compartment voxelization
+        """
+        if voxel_size is None:
+            voxel_size = numpy.array([1, 1, 1], dtype=int)
+
+        if center is None:  # use the middle of the grid
+            center = (self.bounding_box[0] + self.bounding_box[1]) / 2.0
+
+        Z, Y, X = numpy.meshgrid(
+            numpy.arange(z_width),
+            numpy.arange(y_width),
+            numpy.arange(x_width),
+            indexing="ij",
+        )
+        coords = numpy.column_stack(
+            [
+                (X * voxel_size[0] - center[0]).flatten(),
+                (Y * voxel_size[1] - center[1]).flatten(),
+                (Z * voxel_size[2] - center[2]).flatten(),
+            ]
+        )
+
+        mesh = mesh_store.get_mesh(self.gname)
+
+        if hollow:
```

Comment:
do you have a strong sense of the difference in these options? if so it would be good to include in the comments